models/tokens/avalanche_c/tokens_avalanche_c_erc20.sql

```diff
@@ -210,4 +210,8 @@ FROM (VALUES
         ,('0xC55036B5348CfB45a932481744645985010d3A44','WINE',18)
         ,('0x01Af64EF39AEB5612202AA07B3A3829f20c395fd','VINTAGE',18)
         ,('0x501012893eE88976AB8B5289B7a78E307d5d9DCb','NRWL',18)
+        ,('0x09EF821c35B4577f856cA416377Bd2ddDBD3d0C9','MMTH',18)
```

Comment:
https://snowtrace.io/token/0x09EF821c35B4577f856cA416377Bd2ddDBD3d0C9
with decimals 8
```suggestion
        ,('0x09EF821c35B4577f856cA416377Bd2ddDBD3d0C9','MMTH',8)
```